packages/paste-core/components/inline-control-group/src/InlineControlGroup.tsx

```diff
@@ -16,6 +14,7 @@ export interface InlineControlGroupProps
   orientation?: 'vertical' | 'horizontal';
```

Comment:
question: For the horizontal group, is it possible to have the heights match up, even if there's a different amount of content in each item?

Current implementation:

![Screenshot 2023-08-02 at 10 03 38 AM](https://github.com/twilio-labs/paste/assets/3821840/e9213a48-166e-4f99-95e7-9b99651213ff)

Desired implementation:

![image](https://github.com/twilio-labs/paste/assets/3821840/9bc4f342-6cbc-49ff-b8b6-bb057b83418d)